sandbox_template.yaml

```diff
@@ -375,3 +375,34 @@ Resources:
           - ShouldUseMutualTLS
           - true
           - !Ref "AWS::NoValue"
+      AccessLogSettings:
+        DestinationArn: !GetAtt ApiGwAccessLogs.Arn
+        Format: "{ \
+          'requestTime': '$context.requestTime', \
+          'apiId': '$context.apiId', \
+          'accountId': '$context.accountId', \
+          'resourcePath': '$context.resourcePath', \
+          'stage': '$context.stage', \
+          'requestId': '$context.requestId', \
+          'extendedRequestId': '$context.extendedRequestId', \
+          'httpMethod': '$context.httpMethod', \
+          'protocol': '$context.protocol', \
+          'path': '$context.path', \
+          'domainName': '$context.domainName', \
+          'identity': { \
+          'sourceIp': '$context.identity.sourceIp', \
+          'userAgent': '$context.identity.userAgent', \
+          'clientCert':{ \
+          'subjectDN': '$context.identity.clientCert.subjectDN', \
+          'issuerDN': '$context.identity.clientCert.issuerDN', \
+          'serialNumber': '$context.identity.clientCert.serialNumber', \
+          'validityNotBefore': '$context.identity.clientCert.validity.notBefore', \
+          'validityNotAfter': '$context.identity.clientCert.validity.notAfter', \
+          }}}"
+
+  ApiGwAccessLogs:
```

Comment:
can you add a subscription filter as well so that the logs get sent to splunk. You can use one of the other filters in the file as an example